deps/npm/package.json

```diff
@@ -204,7 +204,7 @@
     "sprintf-js": "~1.0.3",
     "standard": "~6.0.8",
     "tacks": "~1.2.2",
-    "tap": "~9.0.3"
+    "tap": "*"
```

Comment:
Yeah, I'm just doing this so I can run the tests with the `-J` option, which is only available in the latest version of tap (that way the tests run 4x faster). This won't land with any changes to `deps/npm`.